website/src/components/Homepage/HomepageCards.tsx

```diff
@@ -0,0 +1,73 @@
+import styles from './cards.module.css';
+import BookIcon from './icons/book.svg';
+import ChatIcon from './icons/chat.svg';
+import DotCircleIcon from './icons/dot-circle.svg';
+import IntegrationsIcon from './icons/integrations.svg';
+import MortarAcademyIcon from './icons/mortar-academy.svg';
+import SdkIcon from './icons/sdk.svg';
+
+const cardsData = [
+    {
+        title: 'Quickstart',
+        description: 'Get up and running with Unleash in less than 5 minutes.',
+        icon: <DotCircleIcon />,
+        href: '/quickstart',
+    },
+    {
+        title: 'Tutorials and Guides',
+        description:
+            'Explore best practices and step-by-step tutorials to help you integrate Unleash into your stack.',
+        icon: <BookIcon />,
+        href: '/feature-flag-tutorials',
```

Comment:
I'll add a tutorials and guides overview page soon, but for now, let's link to the first item in the Use Case tutorials list:

```suggestion
        href: '/feature-flag-tutorials/use-cases/gradual-rollout',
```